### ISSUE TITLE
NUnit1001 compains about DateOnly parameters

### ISSUE THREAD

**Issue by Dreamescaper** [nunit/nunit.analyzers#820]
```csharp
    [TestCase("2024-09-25")]
    [TestCase("2024-09-26")]
    [TestCase("2024-09-27")]
    public async Task TestDateOnly(DateOnly expectedDate)
    {
...
    }
```

NUnit1001 is triggered here, even though it works perfeclty fine if I disable this analyzer.
